cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -70,24 +70,28 @@ public static ClientAndServer getCCServer(int port) throws IOException, URISynta
                 "io.netty.handler.ssl.SslHandler.level=WARNING";
         LogManager.getLogManager().readConfiguration(new ByteArrayInputStream(loggingConfiguration.getBytes(UTF_8)));
 
-        ClientAndServer ccServer = new ClientAndServer(port);
-        return ccServer;
+        ClientAndServer server = new ClientAndServer(port);
+        return server;
     }
 
     private static JsonBody getJsonFromResource(String resource) throws URISyntaxException, IOException {
 
-        URI jsonURI = Objects.requireNonNull(MockCruiseControl.class.getClassLoader().getResource(CC_JSON_ROOT + resource)).toURI();
+        URI jsonURI = Objects.requireNonNull(MockCruiseControl.class.getClassLoader().getResource(CC_JSON_ROOT + resource))
```

Comment:
Out of interest what hard-wrap width are you working to, seems a lot of edits in this PR are hard wraps. I actually think it would be nice to define a project wide one as a style guide.

Reply:
I tend to wrap in logical sections, readbaility is unfortunately a little subjective but the reason I split this was to make clear that we are modifying to the objects, then converting them to a URI
i.e. one clear operation per a line, for things like filters this rule can get a bit more subjective and arguable, but just because a final method is a simple conversioon and only a few letters in length for the method I don't neccessarily think it belongs on the same line as the rest, if that makes sense.

ultimately though this is styling and consistency only has so much value.
Would be open to discussing a hardwrap limit though!